DCOS/fluentd-testing/run_fluentd_tests.ps1

```diff
@@ -0,0 +1,13 @@
+$TESTS_DIR = Split-Path -Parent $MyInvocation.MyCommand.Path
+pushd "$TESTS_DIR"
+$results = Invoke-Pester -PassThru
+$FailedTestTally = $results.FailedCount
+if ($FailedTestTally -eq 0) {
```

Comment:
Please use `$results.FailedCount` directly, you don't need to create another variable `$FailedTestTally`